lib/src/package_name.dart

```diff
@@ -268,8 +268,7 @@ class PackageRange extends PackageName {
     if (!range.includeMin) return this;
     if (range.includeMax) return this;
     if (range.min == null) return this;
-    if (range.max == range.min.nextBreaking.firstPreRelease ||
-        (range.min.isPreRelease && range.max == range.min.nextBreaking)) {
+    if (range.max == range.min.nextBreaking.firstPreRelease) {
```

Comment:
```suggestion
    if (range.max == range.min.nextBreaking.firstPreRelease ||
        (range.min.isPreRelease && range.max == range.min.nextBreaking)) {
        assert(range.includeMax == false);
```

But wouldn't you believe this to be correct if `range.includeMax == false`?

Which is always the case because of the preceeding `if (range.includeMax) return this;`

Reply:
nvm... that still wrong.